parquet/src/test/java/org/apache/iceberg/parquet/TestParquetSchemaUtil.java

```diff
@@ -61,6 +63,128 @@ public class TestParquetSchemaUtil {
           required(116, "dec_38_10", Types.DecimalType.of(38, 10)) // spark's maximum precision
           );
 
+  private static final Schema ICEBERG_SCHEMA =
```

Comment:
Why are we testing this with an Iceberg Schema? Are we unable to craft the same thing with just a StructType like the other tests here?